bodo/pandas/series.py

```diff
@@ -275,18 +303,24 @@ def lower(self):
             name=self._series.name,
             index=index,
         )
+        expr = LazyPlan(
+            "PythonScalarFuncExpression",
+            self._series._plan,
+            (
+                "str.lower",
+                True,  # is_series
+                True,  # is_method
+                (),  # args
+                {},  # kwargs
+            ),
+        )
+        expr.out_schema = new_metadata.to_frame()
         return wrap_plan(
             new_metadata,
             plan=LazyPlan(
-                "LogicalProjectionPythonScalarFunc",
+                "LogicalProjection",
                 self._series._plan,
```

Comment:
other places you use diffrent plan for expression and wrap but here it is it the same?  Document.